app/globals.css

```
@@ -0,0 +1,891 @@
+  color: var(--moloch-800);
+  font-family: var(--font-body), Georgia, serif;
+  font-size: 1rem;
+  line-height: 1.6;
+  font-variant: normal;
+  font-variant-caps: normal;
+  font-feature-settings:
+    "smcp" 0,
+    "c2sc" 0,
+    "pcap" 0,
+    "c2pc" 0;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.1)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)